pkg/config/config.go

```diff
@@ -100,13 +101,18 @@ type OS struct {
 	Environment    map[string]string `json:"environment,omitempty"`
 }
 
+type Node struct {
+	Labels map[string]string `json:"labels,omitempty"`
+}
+
 type HarvesterConfig struct {
 	ServerURL string `json:"serverUrl,omitempty"`
 	Token     string `json:"token,omitempty"`
 
 	OS             `json:"os,omitempty"`
 	Install        `json:"install,omitempty"`
 	RuntimeVersion string `json:"runtimeVersion,omitempty"`
+	Node           `json:"node,omitempty"`
```

Comment:
Harvester config is[ originated from k3os config](https://github.com/rancher/k3os#k3oslabels). Maybe putting under `os`?

Reply:
Didn't know it's from k3os. Will move `labels` to under `os`.